assignments/lambda-classes.js

```diff
@@ -1 +1,72 @@
 // CODE here for your Lambda Classes
+
+class Person {
+
+    constructor(attributes){
+
+      this.name = PersonAttributes.name;
+      this.age = PersonAttributes.age;
+      this.location = PersonAttributes.location;
+      this.gender = PersonAttributes.gender;
+
+    }
+    
+    speak(){
+      return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+}
+
+class Instructors extends Person {
+    constructor (instructorAttributes){
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+
+    demo('subject'){
```

Comment:
When you have functions that take in parameters, you do not put them in quotes. If you want demo to take in a parameter called subject, simply write `demo(subject) {}`